CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - adding narrative_config.yaml spec file for separating narrative indexing.
 
+## [1.9.19] - 2021-02-26
```

Comment:
Can you move this under the [Unreleased] header, then we don't need a version here

Reply:
moved